dts/arm/nxp/nxp_imx7d_m4.dtsi

```diff
@@ -331,6 +331,7 @@
 					       RDC_DOMAIN_PERM_RW))>;
 			label = "PWM_1";
 			status = "disabled";
+			#pwm-cells = <2>;
```

Comment:
The imx changes should be in a separate commit

Reply:
I was wondering where those went ;) [got merged accidentally in here]

Reply:
Dropped the imx change for now